Packs/Base/Scripts/ValidateContent/ValidateContent.py

```diff
@@ -178,7 +177,7 @@ def run_validate(file_path: str, json_output_file: str) -> None:
         os.makedirs(tests_dir)
     with open(f'{tests_dir}/id_set.json', 'w') as f:
         json.dump({}, f)
-    v_manager = ValidateManager(
+    v_manager = OldValidateManager(
```

Comment:
this is not needed at the moment, we need to change this only when we update the docker of `xsoar-tools`.
you can revert this one